projects/onto-search/README.md

```diff
@@ -0,0 +1,24 @@
+# OntoSearch
```

Comment:
Better remove everything here at the moment or update it with proper description.